Assets/Scripts/Dialogue scripts/Dialogue.cs

```diff
@@ -1,197 +1,184 @@
-﻿using UnityEngine;
-using UnityEngine.UI;
-using System.Collections.Generic;
+﻿using System;
 using System.Collections;
 using System.Text;
-using System;
+using UnityEngine;
 
-[RequireComponent(typeof(BoxCollider2D))]
-public class Dialogue : Interacted
+namespace Dialogue_scripts
 {
-    [TextArea] public string[] _textBox; // string array for inputting separate dialogue boxes
-
-    [TextArea] public string[] _hiddenTextBox;
-
-    [SerializeField] private RenderDialogue _pageRender;
-
-    [SerializeField] GameObject rayCast;
-
-    [SerializeField] GameObject dialogueBox;
-
-    public JournalInventoryScript joInvScript;
-
-    [SerializeField] GameObject accessItem;
-
-    [SerializeField] GameObject hiddenItem;
-
-    [SerializeField] GameObject item;
-
-    [SerializeField] private DeathMovement _movement;
-
-    public int _itemClass;
-
-    public int _pageClass;
-
-    BoxCollider2D _myCollider;
-
-    SpriteRenderer _myRenderer;
-
-    public bool isIntro = false;
-
-
-    private void Start()
+    [RequireComponent(typeof(BoxCollider2D))]
```

Comment:
I can see you've implemented this correctly with `GetComponent<BoxCollider2D>` later on, but I figured I should at least remind you the inspector exists for things like this, which allows for more flexible prefab configuration.